docs/speech.rst

```diff
@@ -54,13 +45,19 @@ Functions
     that can be further hand-edited to improve accuracy, inflection and
     emphasis.
 
-.. py:function:: pronounce(phonemes, \*, pitch=64, speed=72, mouth=128, throat=128)
+.. py:function:: pronounce(phonemes, \*, pitch=64, speed=72, mouth=128, throat=128, pin=pin0)
+                 pronounce(phonemes, pin=(pin_speaker, pin0))
```

Comment:
I think this suggest the second version doesn't have any of the other arguments?

Do we also need to add second signatures to all the other methods in speech?

Reply:
> Do we also need to add second signatures to all the other methods in speech?

I think it's good to be explicit. I will do this for speech and music too

Reply:
Done in https://github.com/bbcmicrobit/micropython/pull/692/commits/f5faefa42a44db470fd018c5470232c80a1d4284